packages/web-components/src/components/rux-tree-node/rux-tree-node.scss

```diff
@@ -1,67 +1,34 @@
 :host {
-    position: relative;
-    box-sizing: border-box;
-    width: 100%;
-    padding: 0;
-    margin: 0;
-    user-select: none;
     display: block;
 }
 
 .tree-node {
     color: var(--color-text-primary);
     font-family: var(--font-body-1-font-family);
-    font-size: var(--font-body-1-font-size);
-    font-weight: var(--font-body-1-font-weight);
+    font-size: var(--font-size-base);
```

Comment:
any reason you're not using `--font-control-body-1-font-size`? and `--font-control-body-1-line-height`

Reply:
I think I thought that was the correct token to use now, I'll change the size token back and I'll add the line-height on that class also